Src/WitsmlExplorer.Frontend/components/ContentViews/ServerManager.tsx

```diff
@@ -49,7 +53,12 @@ const ServerManager = (): React.ReactElement => {
         return;
       }
       try {
-        const wells = await WellService.getWells();
+        const [wells, supportedObjects] = await Promise.all([WellService.getWells(), CapService.getCapObjects()]);
+        const updatedVisibility = { ...selectedFilter.objectVisibilityStatus };
```

Comment:
When connecting to a server that does not support an object, and then connecting to another server that does support that object, the object will remain disabled due to reusing `selectedFilter.objectVisibilityStatus` and the following code only disabling objects, but not enabling them back.